src/components/atoms/Checkbox.tsx

```diff
@@ -1,17 +1,14 @@
-// TODO: ISSUE-33 (https://github.com/binarapps/expo-ts-template/issues/33)
-// Remove native-base components when issue is resolved
-import { Text } from 'native-base'
 import { useCallback, useMemo } from 'react'
 import { View, Pressable, StyleSheet } from 'react-native'
 
 import { Icon } from './Icon'
+import { Text } from './Text'
 import { CheckboxProps } from './types'
 
 import { useColorScheme } from '~contexts'
-
 // TODO: ISSUE-33 (https://github.com/binarapps/expo-ts-template/issues/33)
-// Remove `useTheme` hook when issue is resolved
-import { useTheme } from '~hooks'
+// Remove `useNativeBaseTheme` hook when issue is resolved
+import { useNativeBaseTheme } from '~hooks'
```

Comment:
Maybe we can create our own `useTheme` hook that will return colors and rest theme config values 🤔

Reply:
Of course it can be done in the future pull request

Reply:
I did a draft for that. Currently it just returns the `newTheme` object. Next pr will have more of theme 🚀